plugins/module_utils/network/ios/facts/static_routes/static_routes.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, module, subspec="config", options="options"):
 
     def get_static_routes_data(self, connection):
         return connection.get(
-            "sh running-config | section ^ip route|ipv6 route"
+            "sh running-config | section ^ip route|^ipv6 route"
```

Comment:
Please refrain from using the short form of the commands (`show` instead of `sh`). This makes caching useless when turned on.